Formula/hg-git.rb

```diff
@@ -0,0 +1,32 @@
+class HgGit < Formula
+  desc "Git Mercurial Plugin"
+  homepage "http://hg-git.github.io"
+  url "https://bitbucket.org/durin42/hg-git/get/0.8.5.tar.bz2"
+  sha256 "60bf47386fc0c9eef809344ff2be98eaa5add822f70dd25670470d06673fe2c4"
+  head "https://bitbucket.org/durin42/hg-git", :using => :hg
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on :hg
+
+  def install
+    system "python", *Language::Python.setup_install_args(prefix)
+  end
+
+  def caveats; <<-EOS.undent
+    1. Put following lines into your ~/.hgrc
+    2. For more information go to http://hg-git.github.io
+
+        [extensions]
+        hggit=
+
+    EOS
+  end
+
+  test do
+    (testpath/".hgrc").write <<-EOS.undent
+      [extensions]
+      hggit=
+    EOS
+    system "hg", "clone", "git://github.com/schacon/hg-git"
```

Comment:
Test is failing: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/4110/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_hg_git/